quiz.js

```diff
@@ -75,30 +24,21 @@ function middle () {
 	treeObject.height = heightBox.value;						// create a key in treeObject called "height" and add to it the text in the height input box
 	treeObject.character = charBox.value;						// create a key in treeObject called "character" and add to it the text in the character input box
 	buildTree(treeObject)														// run the function "buildTree" with the treeObject object
-	console.log("height:", treeObject.height);
-	console.log("character", treeObject.character);
 }
 
 
 function buildTree (x) {
 
-	var tree = "";															// create a variable "tree" that contains an empty string
-
-
 	if (x.height === "" || x.character === "") {																		// if the value of the key "height" is empty OR
 		alert("Oops! I'm missing information, Please fill out the WHOLE form.");			// if the value of the key "character" is empty
-	}																																								// when this function is called, display this alert
+	}	else {																																							// when this function is called, display this alert
```

Comment:
@NewEvolution I'm not sure why this is, but in my text editor there is just a single space like there should be. But I see that there isn't on here. Why is that?